README.md

```diff
@@ -85,6 +85,14 @@ java -DdebugOperations -cp /path/to/openapi-generator-cli.jar:/path/to/your.jar
 Will, for example, output the debug info for operations.
 You can use this info in the `api.mustache` file.
 
+## Local Test Setup for twilio-go
+To run the unit tests or integration tests locally you need complete the following steps:
+1. Install the [prism CLI](https://meta.stoplight.io/docs/prism/docs/getting-started/01-installation.md) to run the integration tests locally.
+2. Run `make test-docker` from the root directory.
```

Comment:
Doesn't this step do all the testing in Docker? Is the goal here to be able to run the tests outside of Docker?

Reply:
Yes. The goal is to be able to run the tests outside of docker. This step is used to generate the contents of `twilio/rest/oai`. Originally I listed out the commands needed to generate `twilio/rest/oai` but the `./prism.sh` file already includes this logic. So I changed the documentation to run `make test-docker` since it only runs the `./prism` file.